config/settings.py

```diff
@@ -170,7 +178,7 @@
 
 STATIC_URL = "static/"
 STATIC_ROOT = 'static_dist'
-STATICFILES_DIRS = [str(BASE_DIR.joinpath("static"))]
+STATICFILES_DIRS = [str(BASE_DIR.joinpath("static")), os.path.join(BASE_DIR, "components"),]
```

Comment:
I don't think the components directory is meant to be served as static file.
```suggestion
STATICFILES_DIRS = [str(BASE_DIR.joinpath("static"))]
```